### PR TITLE
Migrate planets example to kedro 0.18

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: examples/planets/conf/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+> *Note:* Please do not check in any local configuration to version control.
+
+## Base configuration
+
+The `base` folder is for shared configuration, such as non-sensitive and project-related configuration that may be shared across team members.
+
+WARNING: Please do not put access credentials in the base configuration folder.
+
+## Instructions
```
Comment (shnela) [inline examples/planets/conf/README.md]: Shouldn't we have some text in `Instructions` section?

**Comment by shnela** [comment]
Don't we want keep `examples/planets/conf/local/.gitkeep` not tracked by git?
If so we should use this [.gitignore](https://stackoverflow.com/a/932982/1565454) pattern.

File: examples/planets/docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
+
+
+# planets documentation build
+# configuration file, created by sphinx-quickstart.
+#
+# This file is execfile()d with the current directory set to its
```
Comment (shnela) [inline examples/planets/docs/source/conf.py]: What is `execfile()d`?

**Comment by Raalsky** [comment]
> Don't we want keep examples/planets/conf/local/.gitkeep not tracked by git?
If so we should use this [.gitignore](https://stackoverflow.com/a/932982/1565454) pattern.
In `.gitkeep` there are two related entries:
```
conf/local/**
!conf/local/.gitkeep
```
And I think that solves your question :wink:

File: examples/planets/docs/source/conf.py (hunk 2 above)
Comment (Raalsky) [reply examples/planets/docs/source/conf.py]: Fixed

File: examples/planets/conf/README.md (hunk 1 above)
Comment (Raalsky) [reply examples/planets/conf/README.md]: I've just removed this `README` file. We have some descriptions inside config files.

**Review by shnela — APPROVED** [review]

**Pull request merged**